### ISSUE TITLE
Transparent lines appear when previewing images.

### ISSUE THREAD

**Issue by hakanohi** [ranger/ranger#2049]
#### Runtime Environment

- Operating system and version: Ubuntu 20.04 LTS
- Terminal emulator and version: kitty version 0.15.0
- Python version: 3.8.2
- Ranger version/commit: 1.9.3
- Locale: en_US

#### Current Behavior
Transparent lines appear on the top and the side when previewing image.
![ranger](https://user-images.githubusercontent.com/51394064/87865638-a6913400-c997-11ea-9dc4-955d17b99b0e.gif)

#### Expected Behavior
No random transparent lines when previewing images.
#### Context

Looks weird and is distracting.

#### Steps to reproduce
1. Open kitty and run ranger with image preview enabled.
2. Navigate to a directory with images.
3. Transparent lines appear on the top and the side while navigating and previewing images.

#### Traceback

```

```

**Comment by toonn** [comment]
Are you using the kitty image previewing method?

**Comment by hakanohi** [comment]
Yes I am. Followed the steps mentioned on the ranger wiki.

**Comment by toonn** [comment]
@kovidgoyal, does this sound like a familiar issue involving ncurses and kitty?

**Comment by kovidgoyal** [comment]
The lines when viewing images bug was fixed a long time ago, the OP
seems to be running a very old version of kitty.

**Comment by toonn** [comment]
@hakanohi, could you try running a recent kitty version?

**Comment by hakanohi** [comment]
Sure. I will give it a try.
Also, it's working fine when I switch to bspwm. The gif above was from when I was using Gnome.

**Comment by joaopauloalbq** [comment]
Same issue here with Kitty image preview method

![kitty](https://user-images.githubusercontent.com/9905202/89344774-c8b7d100-d67c-11ea-8640-3839386fb9ff.GIF)


kitty terminal 0.18.1
ranger 1.9.3.4.gd8c363c9-1
Python version: 3.8.3

**Comment by toonn** [comment]
@joaopauloalbq, looks like a different issue, in your case the borders disappear.

**Comment by joaopauloalbq** [comment]
True, in my case, the borders flicker or disappear when changing from one image to another.

**Comment by hakanohi** [comment]
What you are having is issue #1773. I have this issue as well tho.
